config.py

```diff
@@ -934,6 +942,18 @@
                  ('visual/left', 'visual/right'),
                  ('auditory', 'visual')]
     ```
+
+    Contrast the "left" and the "right" regardless of side, and compute an
+    arbitrary contrast with a gradient of weights:
+    ```python
+    conditions = ['auditory/left', 'auditory/right',
```

Comment:
Why do you need both conditions and contrasts?

Reply:
We can only calculate contrasts from conditions that were previously specified in `conditions`; otherwise the indexing we do fails.

Reply:
We don't, that's why I left as a "TODO" just above. So far, contrasts stated `The condition names must be specified in ``conditions`` above.`, and indeed the code used `all_evoked` computed on `conditions` to compute the contrasts. I think this is too restrictive, so I don't do this anymore and instead go back to the `epochs`, and wanted your input on this.

Reply:
agreed. Using epochs seems best !

Reply:
OK, so I removed the `TODOs`, and cleaned (all!) examples of `contrasts`.